src/NuGet/Microsoft.Orleans.Client.nuspec

```diff
@@ -18,9 +18,9 @@
     <copyright>Copyright Microsoft 2015</copyright>
     <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
     <dependencies>
-      <dependency id="Microsoft.Orleans.Core" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansAzureUtils" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansProviders" version="1.0.3" />
+      <dependency id="Microsoft.Orleans.Core" version="$version$" />
```

Comment:
I think we should keep specific minimum versions as before, i.e. 1.0.5 in this case, so that we can publish non-breaking changes to a package as, say, 1.0.6 and not force everyone to update all other packages.